### ISSUE TITLE
Get error when generate models

### ISSUE THREAD

**Issue by lttin0912** [reliese/laravel#26]
I use Laravel 5.4 and have an exists database in MySQ. I want to generate models.
When I run "php artisan code:models" I receive the error: 
  [Doctrine\DBAL\Driver\PDOException]                                          
  SQLSTATE[HY093]: Invalid parameter number: mixed named and positional param  
  eters                                                                                                                                       
  [PDOException]                                                               
  SQLSTATE[HY093]: Invalid parameter number: mixed named and positional param  
  eters    
Please help me. Thanks.

**Comment by CristianLlanos** [comment]
This is weird. Do you happen to know how to replicate this. I've tried with my own database and have no problems.

**Comment by CristianLlanos** [comment]
Do you mind sharing which version of mysql you are using? 🤔

**Comment by lttin0912** [comment]
Yeah, I use MySQL version 5.6.36. I'll try one more time.

**Comment by lttin0912** [comment]
As it would seem, when I run command: "php artisan code:models", it did not use database from .env, it used another database (that is "furnicom"). Do I need to config my database? If yes, please help me config. Thank you!

[Illuminate\Database\QueryException]                                         
  SQLSTATE[HY093]: Invalid parameter number: mixed named and positional param  
  eters (SQL: SHOW FULL COLUMNS FROM `furnicom`.`?:option`)        
  [Doctrine\DBAL\Driver\PDOException]                                          
  SQLSTATE[HY093]: Invalid parameter number: mixed named and positional param  
  eters                                                               
  [PDOException]                                                               
  SQLSTATE[HY093]: Invalid parameter number: mixed named and positional param  
  eters

**Comment by CristianLlanos** [comment]
Do you have a table name `?:option`? 🤔 BTW, you can specify the database schema you are interested in by telling it through the option `--schema`, like this: `php artisan code:models --schema=my_database`. When you don't specify the schema, it uses the default from your `database.php` file which should be filled by the `.env` variables.

**Comment by lttin0912** [comment]
Yes, I config my database in .env and `?:option?` is in another database.
It's very strange, because I config like that but I use "ignasbernotas/laravel-model-generator" package is OK.
